### PR TITLE
update rancher references to k3s-io

### PR DESCRIPTION
**Pull request by briandowns**
Update rancher references to k3s-io.

### CODE REVIEW

**Review by brandond — APPROVED** [review]

File: scripts/manifest [hunk 1]
```diff
@@ -7,7 +7,7 @@ fi
 
 set -e -x
 
-export REPO="rancher/k3s-upgrade"
+export REPO="k3s-io/k3s-upgrade"
```
Comment (cjellick) [inline scripts/manifest:10]: this is meant to be the github repo and not the docker hub repo? because the docker hub repo is still rancher

Comment (cjellick) [reply scripts/manifest:10]: no, looking below it is definitely the docker hub repo

Comment (dweomer) [inline scripts/manifest:10]: We are still only publishing under the rancher org in docker hub. is this the repo for that?

Comment (briandowns) [reply scripts/manifest]: Edited in error. Fixed.

**Review by brandond — APPROVED** [review]

**Review by galal-hussein — APPROVED** [review]
LGTM

**Pull request merged**